### PR TITLE
issue#1 remove leading 0 and space between time and AM/PM

### PR DESCRIPTION
**Pull request by adam-at-brown**
resolves issue #1 - partly

- [x]  remove leading `0` on US format times
- [x]  add space between `AM/PM` and time
- [ ] add function to format date within string - will require new issue

### CODE REVIEW

**Comment by AndrewWJordan** [comment]
Yea I probably should've split up that enhancement request into multiples for this reason. Noted.

File: __tests__/dates.test.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { settingsDateFormat } from "../src/dates";
 describe("settingsDateFormat", () => {
   it("makes a string kebob case", () => {
```
Comment (AndrewWJordan) [inline __tests__/dates.test.ts]: Should this be changed since they aren't all kebab case?

**Review by AndrewWJordan — APPROVED** [review]
LGTM. Left one comment/question.

File: __tests__/dates.test.ts (hunk 1 above)
Comment (adam-at-brown) [reply __tests__/dates.test.ts]: my bad, copy pasta

**Comment by adam-at-brown** [comment]
@AndrewWJordan you should be able to run the test after pulling the latest version of this branch.

**Pull request merged**